polaris-core/src/main/java/org/apache/polaris/core/policy/TransactionalPolicyMappingPersistence.java

```diff
@@ -54,7 +54,7 @@ default void deleteFromPolicyMappingRecordsInCurrentTxn(
   /** See {@link PolicyMappingPersistence#deleteAllEntityPolicyMappingRecords} */
   default void deleteAllEntityPolicyMappingRecordsInCurrentTxn(
       @Nonnull PolarisCallContext callCtx,
-      @Nonnull PolarisEntityCore entity,
+      @Nonnull PolarisBaseEntity entity,
```

Comment:
side note: we should probably review the `PolarisEntityCore` type hierarchy. `PolarisBaseEntity` dominates all child types, so it can probably be folded into `PolarisEntityCore` (or the other way around).